frontend/src/Store/Actions/discoverMovieActions.js

```diff
@@ -235,9 +235,9 @@ export const defaultState = {
   filterBuilderProps: [
     {
       name: 'status',
-      label: 'Status',
+      label: 'Release Status',
```

Comment:
Translate needed

`src\NzbDrone.Core\Localization\Core\en.json` is where you would check for existing keys or add new ones as needed